cmd/kubelet/app/options/options.go

```diff
@@ -87,6 +83,7 @@ type KubeletFlags struct {
 // a kubelet. These can either be set via command line or directly.
 type KubeletServer struct {
 	KubeletFlags
+	ContainerRuntimeOptions
```

Comment:
Also, would it make sense for these to be subordinate to the KubeletFlags object, rather than directly on the KubeletServer, given that they all come in as flags-only?